dojo/settings/settings.dist.py

```diff
@@ -69,6 +69,12 @@
     DD_SECRET_KEY=(str, '.'),
     DD_CREDENTIAL_AES_256_KEY=(str, '.'),
     DD_DATA_UPLOAD_MAX_MEMORY_SIZE=(int, 8388608),  # Max post size set to 8mb
+    DD_SOCIAL_AUTH_TRAILING_SLASH=(bool, False),
+    DD_SOCIAL_AUTH_AUTH0_OAUTH2_ENABLED=(bool, 'False'),
```

Comment:
Please remove the ticks, this is a boolean, and this will fail in your login form to evaluate as you'd expect (see a fix from me earlier on this morning ;-))

Reply:
Fixed thanks I stupidly copy/pasted the previous providers that are wrong (#mentalnote for a next PR).